package.json

```diff
@@ -23,7 +27,9 @@
     "vinyl-source-stream": "^1.1.0",
     "watchify": "^3.7.0"
   },
-  "scripts": {},
+  "scripts": {
+    "test": "karma start --single-run --browsers PhantomJS"
```

Comment:
Let's make it `gulp && karma start --single-run --browsers PhantomJS` so every time we run `npm test` gulp is run before the tests.